src/vm/prestub.cpp

```diff
@@ -261,6 +261,12 @@ PCODE MethodDesc::PrepareInitialCode()
     NotifyGdb::MethodPrepared(this);
 #endif
 
+#ifdef FEATURE_PERFMAP
+    EECodeInfo codeInfo(pCode);
+    size_t sizeOfCode = codeInfo.GetCodeManager()->GetFunctionSize(codeInfo.GetGCInfoToken());
```

Comment:
This lookup is relatively expensive. It should be done only if we actually need to log the method size.